courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>We can get rid of the preview now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done!</p>